client/src/App.js

```diff
@@ -22,7 +23,14 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList} />
+        <Route
+          path="/movies/:id"
+          addToSavedList={this.addToSavedList}
+          render={props => (
+            <Movie {...props} addToSavedList={this.addToSavedList} />
+          )}
+        />
```

Comment:
Part 1:
- [x] Wrap your app with the router.
- [x] Inside your App file add two routes:
- [x] one route for / that loads the MovieList component.
- [x] one route that will take an id parameter after/movies/ (ex: /movies/2, /movies/3 where the id is dynamic). This route should load the Movie component.